app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
       <% end %>
       
       <% if alert.present? %>
-        <%= render 'shared/flash', locals: {message: alert, css_class: "danger"} %>
+        <%= render partial: 'shared/flash', locals: {message: alert, css_class: "danger"} %>
```

Comment:
This would be okay to just have as 'render 'shared/flash' they both work :)

Reply:
including partial is extra, but descriptive nonetheless